Modules/Quotas/test/Quotas.Application.Tests/Tests/Quotas/DeleteIndividualQuota/HandlerTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Application.Tiers.Commands.DeleteQuotaForIdentity;
+using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using Backbone.Modules.Quotas.Domain.Aggregates.Tiers;
+using Enmeshed.BuildingBlocks.Application.Abstractions.Exceptions;
+using Enmeshed.BuildingBlocks.Domain;
+using Enmeshed.UnitTestTools.Extensions;
+using FakeItEasy;
+using FluentAssertions;
+using Microsoft.Extensions.Logging;
+using Xunit;
+using Handler = Backbone.Modules.Quotas.Application.Tiers.Commands.DeleteQuotaForIdentity.Handler;
+using MetricKey = Backbone.Modules.Quotas.Domain.Aggregates.Metrics.MetricKey;
+
+namespace Backbone.Modules.Quotas.Application.Tests.Tests.Quotas.DeleteIndividualQuota;
+public class HandlerTests
+{
+    [Fact]
+    public async Task Delete_individual_quota()
+    {
+        // Arrange
+        var tierId = new TierId("SomeTierId");
+        var identityAddress = "some-identity-address";
+        var identity = new Identity("some-identity-address", tierId);
+        var createdQuota = identity.CreateIndividualQuota(MetricKey.NumberOfSentMessages, 5, QuotaPeriod.Month);
+        var command = new DeleteQuotaForIdentityCommand(identityAddress, createdQuota.Id);
```

Comment:
I disagree since the test focuses on the deletion of individual quotas, hence the creation of identities and their respective quotas should qualify as an arrangement, no?

Reply:
But.. it's the DELETEQuotaForIdentityCommand? ;-)

Reply:
> we've followed this convention for the other hundreds of tests

Did we? I never noticed that. But if that's the case, I'm okay with you leaving it as it is for now.